packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -854,34 +740,14 @@ class InspectorController extends DisposableController
     return null;
   }
 
-  void syncSelectionHelper({
-    required bool maybeRerootDetailsTree,
-    required RemoteDiagnosticsNode? selection,
-    required RemoteDiagnosticsNode? detailsSelection,
-  }) {
+  void syncSelectionHelper({required RemoteDiagnosticsNode? selection}) {
     if (selection != null) {
       if (selection.isCreatedByLocalProject) {
         _navigateTo(selection);
       }
     }
-    if (detailsSubtree || details == null) {
-      if (selection != null) {
-        var toSelect = selectedNode.value;
 
-        while (toSelect != null && toSelect.diagnostic!.isProperty) {
-          toSelect = toSelect.parent;
-        }
-
-        if (toSelect != null) {
-          final diagnosticToSelect = toSelect.diagnostic!;
-          unawaited(diagnosticToSelect.setSelectionInspector(true));
-        }
-      }
-    }
-
-    if (maybeRerootDetailsTree) {
-      _showDetailSubtrees(selection, detailsSelection);
-    } else if (selection != null) {
+    if (selection != null) {
       // We can't rely on the details tree to update the selection on the server in this case.
```

Comment:
obsolete comment. Maybe search for "details tree" and "summary tree" to clean up the breadcrumbs of this concept throughout comments too?

Reply:
done! helped me find more code to delete